### ISSUE TITLE
Add a button near a video to "Play to Kodi' for several services

### ISSUE THREAD

**Issue by tbitowt** [khloke/play-to-xbmc-chrome#78]
Right now, to play a video on kodi, I need to:
a) Right click and select 'Play to Kodi'
b) Click on an extension menu on right upper corner and select 'Play'

It would be very useful, to add a dedicated button to a several services (like youtube) to 'Play this', 'Play playlist' etc.

It's an enhancement proposition

**Comment by khloke** [comment]
Hi tibtowt,

I really appreciate your suggestion, however this is something that I have to reject. This is something I set out to avoid right from the beginning of the project. Before I starting working on this, there were already a number of extensions in the market that does this, and the common problem among all these extensions is that, whenever YouTube decides to change its webpage, these extensions stop working, and the developer will then have to start working on changing the code in their extension to work with the new webpage, this presents 2 problems: (1) It takes time, meaning the extension will not work until fixed and (2) High maintenance for the extension.

When I realized that the extensions that I was using at the time was breaking every few months and only starts working after a few days, or even in some cases, never fixed, I set out to make my own that would get around this problem. What I did ended up as Play to Kodi, where the button is in the extension, and it reads the URL of the page rather than the page itself, meaning that YouTube can change their webpage any way they like, and the extension keeps working. Reading the URL is great because it is highly unlikely for YouTube to change that because it will break the links people have shared all over the internet, and because the button lives outside of the page, it doesn't get affected. I know many people may not agree with me, but I believe this is an advantage that is a victim of its own success and people have taken it for granted without even realizing it.

TL;DR: Sorry. Won't implement.

Cheers,
KH
